components/topic-overview/TopicOverviewNav.tsx

```diff
@@ -35,11 +37,10 @@ export const TopicOverviewNav = ({selected, setSelected}:Props) => {
     const scrollToCustomPosition = (id:string) => {
         setSelected(id)
         const targetElement = document.getElementById(id);
-        const offset = -50; // Adjust this value to the desired scroll height
 
         if (targetElement) {
             const topPosition = targetElement.getBoundingClientRect().top;
-            window.scrollTo({ top: topPosition + offset, behavior: 'smooth' });
```

Comment:
just fyi - i had some issues with scrollTo not being super consistent across different browsers (although that was for live updating position with the audioPlayer prototype) so something to keep in mind when testing